waiter/test/waiter/async_request_test.clj

```diff
@@ -326,6 +326,50 @@
           (complete-async-request-fn :success)
           (is (nil? (get @async-request-store-atom request-id))))))))
 
+(deftest test-post-process-async-request-response-sanitized-check-interval
+  (let [{:keys [host port] :as instance} {:host "www.example.com", :port 1234}
+        router-id "my-router-id"
+        service-id "test-service-id"
+        metric-group "test-metric-group"
+        backend-proto "http"
+        async-request-store-atom (atom {})
+        request-id "request-2394613984619"
```

Comment:
Where did this long number come from? If it's not actually important, maybe just `123` or something would be better?